frappe/public/less/quill.less

```diff
@@ -131,13 +131,28 @@
 
 .ql-editor ol {
 	padding-left: 2.5em;
+  padding-right: 2.5em;
 }
 
 .ql-editor li[data-list=ordered] {
 	list-style-type: decimal;
 	padding-left: 0;
+  padding-right: 0;
 
 	&::before {
 		content: none;
 	}
 }
+
+// RTL alignment
+.control-input {
+  direction: ltr;
+}
+
+.ql-container {
+  direction: ltr;
+}
+
+.page-body {
+  overflow-x: visible;
+}
```

Comment:
We can remove `overflow-x: hidden;` form `.page-body` in `page.less` it's the same.

Reply:
@afarran3 It's a better idea to leave that piece of code as it is (since `overflow-x: hidden` has been added intentionally) and do the change suggested above instead! Would you mind if I go ahead and push the suggested changes myself?